openassessment/xblock/openassessmentblock.py

```diff
@@ -404,6 +404,19 @@ def should_use_user_state(self, upload_urls):
             and self.user_state_upload_data_enabled() \
             and self.file_upload_response
 
+    def should_get_all_files_urls(self, upload_urls):
```

Comment:
can add a little info in doc string about `upload_urls` argument?

Reply:
ok